detection_rules/kbwrap.py

```diff
@@ -230,6 +265,39 @@ def kibana_export_rules(
 
             exceptions.append(exception)
 
+    # Parse action connector results from API return
+    action_connectors = []
+    if export_action_connectors:
```

Comment:
Not sure if we should do this here or in the DAC feature branch but these big if statements could probably be broken into smaller methods.

That way the code can be reused here and in the next cli command in main.py. They look similar enough at least.

Reply:
Agreed, I think we should do it in the DAC Feature branch or the Autogen custom schema branch (https://github.com/elastic/detection-rules/pull/3949), as to not have too much diff created here.